### ISSUE TITLE
Only WCAG2AA

### ISSUE THREAD

**Issue by markwbrown** [pa11y/pa11y-ci#90]
Pa11y has options for Section508, WCAG2A & WCAG2AA. I don't see anywhere in the docs or code for Pa11y-ci to use the other options

**Comment by aarongoldenthal** [comment]
You can set this via the [`standard`](https://github.com/pa11y/pa11y#standard-string) configuration setting.

It can be set globally:

```json
{
    "defaults": {
        "standard": "Section508"
    },
    "urls": [
        "https://www.w3.org/WAI/demos/bad/before/home.html", 
        "https://www.w3.org/WAI/demos/bad/before/news.html"
    ]
}
```

or per url if you need different standards applied to each

```json
{
    "urls": [
        "https://www.w3.org/WAI/demos/bad/before/home.html", 
        {
            "url": "https://www.w3.org/WAI/demos/bad/before/news.html",
            "standard": "Section508"
        }
    ]
}
```

**Comment by shreyarajwade** [comment]
Based on the pa11y-ci documentation:

By default, Pa11y CI looks for a config file in the current working directory, named .pa11yci. This should be a JSON file.

You can use the --config command line argument to specify a different file, which can be either JSON or JavaScript. The config files should look like this:
`{
    "urls": [
        "http://pa11y.org/",
        "http://pa11y.org/contributing"
    ]
}`

I added the following code in .pally-ci file:
`{
	"defaults": {
    	"standard": "WCAG2A"
	}
}
`
But after running pa11y-ci through terminal using the command: 
`pa11y-ci --sitemap https://www.example.com/sitemap.xml --json > path/to/file/result.json`, 
it is still providing result for standard WCAG2AA
I even added the standard defaults.json and defaults-overridden.json, it still provides result for WCAG2AA
Is there some other file where the standard needs to be specified?

**Comment by josebolos** [comment]
Thanks for creating this issue. It looks like this is now fixed, feel free to reopen the issue if not.
